README.md

```diff
@@ -1,50 +1,65 @@
 
 
 # Usage
-Easiest way to run things is to make some compound executables in CLion that boot everything you need. Make sure you have grsim or ssl-vision running and are listening to the right ports.
+First, clone this repository. We recommend using an SSH key if you are able to do so, as this immediately authorizes your future pushes to the repositories. 
+```
+git clone --recurse-submodule git@github.com:RoboTeamTwente/roboteam_suite.git
+```
+Now, you can open the roboteam_suite project with an IDE of your choice. We recommed CLion, as it has a lot of very helpful features and you can get a student license for it. 
+
+Make sure you have CMake installed. Go to the CMakeLists.txt and click on it. CLion will generate a pop-up saying "project files not loaded" in blue in the top right corner. Clicking it will have CMake load the project. This means it generates instructions on how the files should be compiled in order to produce the executables specified in the CMakeLists.txt file. You can now compile the targets!
 
+The easiest way to run the application is to make some [compound executables](https://www.jetbrains.com/help/clion/run-debug-configuration.html#config-folders) in CLion that boot everything you need. To run 1 team in GRSim, the simulator, you will need to add roboteam_ai, roboteam_world, and roboteam_robothub. 
+Make sure you have GRSim or ssl-vision running and are listening to the right ports. In general, the GRSim vision multicast port should be set to 10006.
 
-running one team:
+## Auxiliary Software
+It is recommended to use the game controller to control the gamestates. You can do so by downloading the latest binary release of the [SSL-Game-Controller](https://github.com/RoboCup-SSL/ssl-game-controller) repo, and executing it. For example, when downloaded to the Downloads folder:
 ```
-./kalman_world
-./roboteam_ai
-./roboteam_robothub 
+~/Downloads/ssl-game-controller_v1.2.0_linux_amd64
 ```
-running two teams:
+You might need to [chmod](https://www.howtoforge.com/tutorial/linux-chmod-command/) the file to make it executable. The AI client should now listen to the commands sent by the GameController. 
+
+Also, make sure you have installed GRSim using the instructions found on the [RoboTeam fork of GRSim](https://github.com/RoboTeamTwente/grSim). Now execute the grsim binary from the command line. This usually looks like this:
 ```
-./kalman_world
-./roboteam_ai
-./roboteam_robothub 
-./roboteam_ai 1
-./roboteam_robothub 1
+/directory_you_cloned_grsim/grSim/bin/grsim
 ```
+## Making the executables: 
+
+Make sure the vision multicast port is set to 10006
+
+### One Team: 
+Make the compound executable shown below:
+<img src="https://github.com/RoboTeamTwente/roboteam_suite/blob/RobotJesse-patch-1/readme_images/run_two_teams.png?raw=true" width="800" height="500"><br>
```

Comment:
Uhhm, there's a markdown way to do this.
```suggestion
![](https://github.com/RoboTeamTwente/roboteam_suite/blob/RobotJesse-patch-1/readme_images/run_two_teams.png?raw=true)
```

Reply:
That doesn't let you specify size and it makes the image humongous

Reply:
Fair enough, mb